src/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="./styles/styles.css" />
+
+    <title>Bicycle Shop</title>
+  </head>
+
+  <body>
+    <header class="header wrapper">
+        <a class="header__logo">
+          <img src="assets/images/icons/logo.svg" alt="Logo Bicycle Shop" />
+        </a>
+
+
+        <h1 class="header__title">Paixão pela bike, sede por desafios</h1>
+
+        <a class="header__cart">
+          <img src="assets/images/icons/cart.svg" alt="Carrinho" />
+        </a>
+    </header>
+
+    <main class="corpo">
```

Comment:
Se a classe corpo não foi utilizada em nenhum outro lugar, não seria melhor retirá-la?
Se for colocada a classe wrapper aqui seria mais interessante, já que todo resto esta dentro da main.